lgc/patch/RegisterMetadataBuilder.cpp

```diff
@@ -1477,19 +1475,15 @@ void RegisterMetadataBuilder::setIaMultVgtParam() {
       usePrimitiveId = gsBuiltInUsage.primitiveId;
     }
 
-    if (isIaMultVgtParamPiped) {
-      auto iaMultVgtParamPiped =
-          getGraphicsRegNode()[Util::Abi::GraphicsRegisterMetadataKey::IaMultiVgtParamPiped].getMap(true);
-      iaMultVgtParamPiped[Util::Abi::IaMultiVgtParamPipedMetadataKey::SwitchOnEoi] = usePrimitiveId;
+    auto iaMultVgtParam = getGraphicsRegNode()[Util::Abi::GraphicsRegisterMetadataKey::IaMultiVgtParam].getMap(true);
+    iaMultVgtParam[Util::Abi::IaMultiVgtParamPipedMetadataKey::SwitchOnEoi] = usePrimitiveId;
+    if (m_isNggMode || (m_gfxIp.major == 10 && !m_isNggMode)) {
```

Comment:
Maybe we can move the metadata definition of `Util::Abi::GraphicsRegisterMetadataKey::IaMultiVgtParamPiped`.

Reply:
Fixed.